moneyd-gui fails to clone

The instance doesn't appear to have the credentials necessary to install the package.

```
aws_instance.web (remote-exec): ----------
aws_instance.web (remote-exec):           ID: gui-clone
aws_instance.web (remote-exec):     Function: git.latest
aws_instance.web (remote-exec):         Name: https://github.com/sharafian/moneyd-gui.git
aws_instance.web (remote-exec):       Result: False
aws_instance.web (remote-exec):      Comment: Clone failed: Cloning into '/srv/gui'...
aws_instance.web (remote-exec):               /srv/gui/.git: Permission denied
aws_instance.web (remote-exec):      Started: 17:56:35.307534
aws_instance.web (remote-exec):     Duration: 524.003 ms
aws_instance.web (remote-exec):      Changes:
aws_instance.web (remote-exec): ----------
aws_instance.web (remote-exec):           ID: gui-install
aws_instance.web (remote-exec):     Function: cmd.run
aws_instance.web (remote-exec):         Name: npm install --unsafe-perm --json --production
aws_instance.web (remote-exec):       Result: True
aws_instance.web (remote-exec):      Comment: Command "npm install --unsafe-perm --json --production" run
aws_instance.web (remote-exec):      Started: 17:56:35.831878
aws_instance.web (remote-exec):     Duration: 886.386 ms
aws_instance.web (remote-exec):      Changes:
aws_instance.web (remote-exec):               ----------
aws_instance.web (remote-exec):               pid:
aws_instance.web (remote-exec):                   12292
aws_instance.web (remote-exec):               retcode:
aws_instance.web (remote-exec):                   0
aws_instance.web (remote-exec):               stderr:
aws_instance.web (remote-exec):                   npm WARN saveError ENOENT: no such file or directory, open '/srv/gui/package.json'
aws_instance.web (remote-exec):                   npm WARN saveError EACCES: permission denied, open '/srv/gui/package-lock.json.4184670851'
aws_instance.web (remote-exec):                   npm WARN enoent ENOENT: no such file or directory, open '/srv/gui/package.json'
aws_instance.web (remote-exec):                   npm WARN gui No description
aws_instance.web (remote-exec):                   npm WARN gui No repository field.
aws_instance.web (remote-exec):                   npm WARN gui No README data
aws_instance.web (remote-exec):                   npm WARN gui No license field.
aws_instance.web (remote-exec):
aws_instance.web (remote-exec):
aws_instance.web (remote-exec):                   ┌──────────────────────────────────────────────────────────┐
aws_instance.web (remote-exec):                   │                 npm update check failed                  │
aws_instance.web (remote-exec):                   │           Try running with sudo or get access            │
aws_instance.web (remote-exec):                   │           to the local update config store via           │
aws_instance.web (remote-exec):                   │ sudo chown -R $USER:$(id -gn $USER) /home/ubuntu/.config │
aws_instance.web (remote-exec):                   └──────────────────────────────────────────────────────────┘
aws_instance.web (remote-exec):               stdout:
aws_instance.web (remote-exec):                   {
aws_instance.web (remote-exec):                     "added": [],
aws_instance.web (remote-exec):                     "removed": [],
aws_instance.web (remote-exec):                     "updated": [],
aws_instance.web (remote-exec):                     "moved": [],
aws_instance.web (remote-exec):                     "failed": [],
aws_instance.web (remote-exec):                     "warnings": [
aws_instance.web (remote-exec):                       "ENOENT: no such file or directory, open '/srv/gui/package.json'\nThis is related to npm not being able to find a file.\n",
aws_instance.web (remote-exec):                       "gui No description",
aws_instance.web (remote-exec):                       "gui No repository field.",
aws_instance.web (remote-exec):                       "gui No README data",
aws_instance.web (remote-exec):                       "gui No license field."
aws_instance.web (remote-exec):                     ],
aws_instance.web (remote-exec):                     "elapsed": 54
aws_instance.web (remote-exec):                   }
aws_instance.web (remote-exec): ----------
aws_instance.web (remote-exec):           ID: gui-start
aws_instance.web (remote-exec):     Function: cmd.run
aws_instance.web (remote-exec):         Name: pm2 start index.js
aws_instance.web (remote-exec):       Result: False
aws_instance.web (remote-exec):      Comment: Command "pm2 start index.js" run
aws_instance.web (remote-exec):      Started: 17:56:36.718635
aws_instance.web (remote-exec):     Duration: 275.356 ms
aws_instance.web (remote-exec):      Changes:
aws_instance.web (remote-exec):               ----------
aws_instance.web (remote-exec):               pid:
aws_instance.web (remote-exec):                   12309
aws_instance.web (remote-exec):               retcode:
aws_instance.web (remote-exec):                   1
aws_instance.web (remote-exec):               stderr:
aws_instance.web (remote-exec):                   [PM2][ERROR] script not found : /srv/gui/index.js
aws_instance.web (remote-exec):                   script not found : /srv/gui/index.js
aws_instance.web (remote-exec):               stdout:
aws_instance.web (remote-exec):                   ┌───────────┬────┬──────┬───────┬────────┬─────────┬────────┬─────┬───────────┬────────┬──────────┐
aws_instance.web (remote-exec):                   │ App name  │ id │ mode │ pid   │ status │ restart │ uptime │ cpu │ mem       │ user   │ watching │
aws_instance.web (remote-exec):                   ├───────────┼────┼──────┼───────┼────────┼─────────┼────────┼─────┼───────────┼────────┼──────────┤
aws_instance.web (remote-exec):                   │ connector │ 0  │ fork │ 12241 │ online │ 0       │ 1s     │ 0%  │ 70.6 MB   │ ubuntu │ disabled │
aws_instance.web (remote-exec):                   └───────────┴────┴──────┴───────┴────────┴─────────┴────────┴─────┴───────────┴────────┴──────────┘
aws_instance.web (remote-exec):                    Use `pm2 show <id|name>` to get more details about an app
```

--- a/tier-1/salt/gui.sls
+++ b/tier-1/salt/gui.sls
@@ -1,14 +1,14 @@
 gui-dir:
   file.directory:
     - name: /srv/gui
-    - runas: ubuntu
+    - user: ubuntu
 
 gui-clone:
   git.latest:
     - name: https://github.com/sharafian/moneyd-gui.git
     - target: /srv/gui
     - rev: master
-    - runas: ubuntu
+    - user: ubuntu
     - require:
       - pkg: build-essential
       - pkg: nodejs

